monkey/monkey_island/cc/ui/src/components/map/MapOptions.js

```diff
@@ -41,9 +41,12 @@ export const basic_options = {
   physics: {
     barnesHut: {
       gravitationalConstant: -120000,
-      avoidOverlap: 0.5
+      avoidOverlap: 0.5,
+      springLength: 100,
```

Comment:
Ideally this should be dynamic, based on how many exploitations we have, but probably and overkill for our default scenario.